com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java

```diff
@@ -166,6 +166,7 @@
     public static final String VAR_HOOK_FOR_EXECUTION = "hookForExecution";
     public static final String VAR_FILE_ENTRIES = "fileEntries";
     public static final String VAR_APPS_TO_RENAME = "appsToRename";
+    public static final String VAR_PASSED_TESTING_PHASE = "passedTestingPhase";

```

Comment:
The name of the variable is too specific. Why should the `CreateOrUpdateAppStep` know about the existence of a testing phase? What we want to tell the step via this variable is whether it should skip the bindings or not.

Reply:
Changed to "shouldSkipServiceRebinding"